components/contact-details.tsx

```diff
@@ -69,65 +33,31 @@ const DescriptionItem = (props: DescriptionItemProps) => {
         {typeof value == "string" &&
           value.length > 0 &&
           props.withCopyButton && <CopyButton text={stripTags(value)} />}
-        <ReportButton onClick={props.onClick} />
       </dd>
     </div>
   );
 };
 
-export function ContactDetails({ contact, provinceName }: ContactDetailsProps) {
-  const _reportButtonHandler = () => {
-    reportButtonHandler({ contact, provinceName });
-  };
-
+export function ContactDetails({ contact }: ContactDetailsProps) {
   return (
     <div className="bg-white shadow overflow-hidden rounded-md">
       <dl className="divide-y divide-gray-200">
-        <DescriptionItem
-          label="Penyedia"
-          onClick={_reportButtonHandler}
-          value={contact.penyedia}
-        />
-        <DescriptionItem
-          label="Keterangan"
-          onClick={_reportButtonHandler}
-          value={contact.keterangan}
-        />
-        <DescriptionItem
-          label="Lokasi"
-          onClick={_reportButtonHandler}
-          value={contact.lokasi}
-        />
-        <DescriptionItem
-          label="Kontak"
-          onClick={_reportButtonHandler}
-          value={contact.kontak}
-          withCopyButton
-        />
-        <DescriptionItem
-          label="Alamat"
-          onClick={_reportButtonHandler}
-          value={contact.alamat}
-          withCopyButton
-        />
-        <DescriptionItem
-          label="Tautan"
-          onClick={_reportButtonHandler}
-          value={contact.link}
-        />
+        <DescriptionItem label="Penyedia" value={contact.penyedia} />
+        <DescriptionItem label="Keterangan" value={contact.keterangan} />
+        <DescriptionItem label="Lokasi" value={contact.lokasi} />
+        <DescriptionItem label="Kontak" value={contact.kontak} />
+        <DescriptionItem label="Alamat" value={contact.alamat} />
```

Comment:
```suggestion
        <DescriptionItem label="Kontak" value={contact.kontak} withCopyButton />
        <DescriptionItem label="Alamat" value={contact.alamat} withCopyButton />
```

Please keep the `withCopyButton` props intact. 😅

Reply:
Oops, it seems I forgot to add it. I will change soon. 😅